src/index.html

```diff
@@ -3,12 +3,61 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width,
+    user-scalable=no,
+    initial-scale=1.0,
+    maximum-scale=1.0,
+    minimum-scale=1.0"
+    >
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="main__header">
+    <div class="container">
+      <a href="https://www.moyo.ua/" class="logo">
+        <img src="./images/logo.png"
+        alt="logo"
+        class="logo_img"
+        >
```

Comment:
переносятся все аргументы + отступы
```suggestion
        <img
          src="./images/logo.png"
          alt="logo"
          class="logo_img"
        >
```